src/features/services/queries.ts

```diff
@@ -24,43 +30,31 @@ interface TokenInfo {
 
 export function useTokenList() {
   const [network] = useNetworkContext();
-  const query = useQuery<LedgerInfoResponse, Error>({
-    queryKey: ["ledger.tokens", network?.url],
-    queryFn: async () => await network?.ledger.info(),
-    enabled: !!network?.url,
-  });
-  const data = query.data?.symbols;
-  return { ...query, data };
+
+  return useQuery<LedgerInfoResponse, Error>(
+    ["tokens"],
```

Comment:
How cache invalidation works with `react-query`: This list of keys is semi-hierarchical, so you can say `["tokens"]` or `["tokens", address]` and then both will clear when you clear `["tokens"]`.